README.md

```diff
@@ -54,6 +54,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |21| Personal Tab with Nav-Bar menu | Add multiple actions to the upper right in Nav-Bar and build an overflow menu for extra actions in an app. ||[View][tab-navbar-menu#ts]|
```

Comment:
Build and deploy pipeline is missing for this sample and also ESBuild configuration is missing.

Reply:
Fixed

Reply:
We need to check and add deploy pipeline as well.